### PR TITLE
Final Model - Extraction of Pre-processed Data from VM

### PR DESCRIPTION
**Pull request by sc4789**
Sana & Preethi - Working on final model with fine tuning and hyperparameter tuning

### CODE REVIEW

**Review by stevenwinnick — COMMENTED** [review]
Looking great!

File: Final Model.ipynb [hunk 1]
```diff
@@ -1035,64 +836,251 @@
...
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "len(X_lyrics)"
+   ]
   },
   {
    "cell_type": "code",
```
Comment (tinghui2012) [inline Final Model.ipynb:2156]: In the following cell (execution 36), why do you set num_genres = 3?

Comment (K-Wangaroo) [reply Final Model.ipynb:2156]: same question, there appeared to be >3 genres in the raw dataset i was looking at?

Comment (sc4789) [reply Final Model.ipynb:2156]: i think this was a prior version - before the midterm i believe

**Pull request merged**